src/mon/OSDMonitor.cc

```diff
@@ -982,6 +995,38 @@ void OSDMonitor::create_pending()
       pending_inc.new_nearfull_ratio = g_conf->mon_osd_nearfull_ratio;
     }
   }
+  creating_pgs = update_pending_creatings(0);
+}
+
+creating_pgs_t OSDMonitor::update_pending_creatings(epoch_t last_scan_epoch)
+{
```

Comment:
Should we only start doing this when 
monmap.get_required_features().contains_all(ceph::features::mon::FEATURE_LUMINOUS)
?